packages/server/tests/acceptance.spec.ts

```diff
@@ -487,4 +493,55 @@ describe('RPC Server Integration Tests', async function () {
         const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getTransactionCount', [ethCompAccountEvmAddr3, mirrorContractDetails.block_number]);
         expect(res.data.result).to.be.equal('0x2');
     });
+
+    it('should execute "eth_getTransactionReceipt" for hash of legacy transaction', async function () {
+        const signedTx = await utils.signRawTransaction({
+            ...defaultLegacyTransactionData,
+            to: mirrorContract.evm_address,
+            nonce: ethCompNonce
+        }, ethCompPrivateKey3);
+        const txRes = await utils.callSupportedRelayMethod(this.relayClient, 'eth_sendRawTransaction', [signedTx]);
+        expect(txRes.data.result).to.be.not.be.null;
+        ethCompNonce++;
+
+        await utils.sleep(3000);
+
+        const submittedLegacyTransactionHash = txRes.data.result;
+        const res = await utils.callSupportedRelayMethod(this.relayClient, 'eth_getTransactionReceipt', [submittedLegacyTransactionHash]);
+
+        utils.assertReceiptNotNull(res.data.result);
+
+        expect(res.data.result.status).to.eq('0x1');
```

Comment:
Instead of having duplicated checks, why dont' we create `assertTransactionReceipt(transactionRequest, transactionReceipt)` and have all the checks there.